include/RAJA/exec-cuda/scan_cuda.hxx

```diff
@@ -103,6 +103,16 @@ struct minimum : public ::thrust::minimum<T> {
 
 }  // closing brace for cuda_scan namespace
 
+namespace scan
+{
+
+template <typename T>
+struct defaults <cuda_exec, T> {
```

Comment:
This idea of defaults is quite interesting, I'm wondering whether it could be expanded in other places. I'll keep thinking about it and ping @rhornung67 , we might be able to provide sane defaults in other places, and I like this way of doing so